emcc.py

```diff
@@ -1538,6 +1538,8 @@ def parse_passes(string):
               shared.Settings.BYSYNCIFY_IMPORTS += ['invoke_*']
             if shared.Settings.BYSYNCIFY_IMPORTS:
               passes += ['--pass-arg=bysyncify-imports@%s' % ','.join(['env.' + i for i in shared.Settings.BYSYNCIFY_IMPORTS])]
+          if shared.Settings.BINARYEN_IGNORE_IMPLICIT_TRAPS:
+            passes += ['--ignore-implicit-traps']
```

Comment:
Won't this end up running the pass twice during do_binaryen?  Once in asm2wasm and then again in wasm-opt?     Perhaps that is intentional?

Reply:
This is more of a flag that determines how the optimizer runs. So we need to pass it to all invocations of the optimizer, so that we ignore implicit traps in all of them.